frontend/src/index.css

```diff
@@ -26,3 +26,7 @@ scrollbar-thumb {
   background-color: rgba(127, 125, 125, 0.4);
   border-radius: 10px;
 }
+
+.cover {
+  object-fit: cover;
+}
```

Comment:
We prefer to add only those CSS here which we want to either apply globally or have requirement in multiple places. You can add this object-fit property using `style` attribute in `<img />` itself.